apps/backoffice-v2/src/pages/MerchantMonitoringUploadMultiple/hooks/useMerchantMonitoringUploadMultiplePageLogic/useMerchantMonitoringUploadMultiplePageLogic.tsx

```diff
@@ -0,0 +1,62 @@
+import { z } from 'zod';
+import { useNavigate } from 'react-router-dom';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import csvContent from './batch-report-template.csv?raw';
+import { useLocale } from '@/common/hooks/useLocale/useLocale';
+import { CreateBatchBusinessReportSchema } from '@/pages/MerchantMonitoringUploadMultiple/create-batch-business-report-schema';
+import { useCreateBusinessReportBatchMutation } from '@/domains/business-reports/hooks/mutations/useCreateBusinessReportBatchMutation/useCreateBusinessReportBatchMutation';
+import { useCallback, useMemo } from 'react';
+
+export const useMerchantMonitoringUploadMultiplePageLogic = () => {
+  const form = useForm({
+    defaultValues: {
+      merchantSheet: undefined,
+    },
+  } as { defaultValues: { merchantSheet: File | undefined } });
```

Comment:
Usually you'd use the type of the same zod schema you use for the validation here.
```suggestion
  const form = useForm<{ merchantSheet: File | undefined }>({
    defaultValues: {
      merchantSheet: undefined,
    },
  });
```